src/platform/landing-pages/dev-template.ejs

```diff
@@ -66,76 +66,84 @@
       ></div>
 
       <!-- Tablet/Desktop Layout -->
-      <div id="legacy-header" class="vads-u-display--none">
-        <!-- USA website banner -->
-        <div class="va-notice--banner">
-          <div class="va-notice--banner-inner">
-            <div class="usa-banner">
-              <div class="usa-accordion">
-                <div class="usa-banner-header">
-                  <div class="usa-grid usa-banner-inner">
-                    <img src="/img/tiny-usa-flag.png" alt="U.S. flag">
-                    <p>An official website of the United States government</p>
-                    <button id="usa-banner-toggle" class="usa-accordion-button usa-banner-button" aria-expanded="false" aria-controls="gov-banner">
-                      <span class="usa-banner-button-text">Here’s how you know</span>
-                    </button>
-                  </div>
-                </div>
-                <div class="usa-banner-content usa-grid usa-accordion-content" id="gov-banner" aria-hidden="true">
-                  <div class="usa-banner-guidance-gov usa-width-one-half">
-                    <img class="usa-banner-icon usa-media_block-img" src="/img/icon-dot-gov.svg" alt="Dot gov">
-                    <div class="usa-media_block-body">
-                      <p>
-                        <strong>The .gov means it’s official.</strong>
-                        <br>
-                        Federal government websites often end in .gov or .mil. Before sharing sensitive information, make sure you're on a federal government site.
-                      </p>
+      <% if (typeof noHeader !== 'undefined' && noHeader) { %>
```

Comment:
Is there ever an instance where we wouldn't want any sort of header rendered? Or, is the intention here to let content-build render in the minimal header? On first pass it looks like this could be handled by just one flag in an apps registry.json entry. 

It's hard for me to read `ejs` in github, but it looks like users could stumble into unintentionally rendering 2 headers if they forget to add `noHeader: true` but include `customHeader: true`, or do something like `noHeader: false` with `customHeader: true`

Reply:
I added this noHeader check to match the functionality within content-build, which currently has a noHeader flag. So, yes, `noHeader: true` and `customHeader: true` would both need to be present to prevent rendering two headers, unless whoever implements this in the future re-writes the control flow logic.